presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -259,6 +259,8 @@ void PrestoExchangeSource::abortResults() {
               "Abort results failed: {}, path {}", statusCode, self->basePath_);
           LOG(ERROR) << errMsg;
           onFinalFailure(errMsg, queue);
+        } else {
+          self->abortResultsSucceeded_.store(true);
```

Comment:
Why do we need to call store() and load() for atomics?
Wouldn't simple = operators do the job?
It clatters the code.